engine/access/rpc/backend/backend_stream_transactions.go

```diff
@@ -57,11 +59,12 @@ func (b *backendSubscribeTransactions) SubscribeTransactionStatuses(
 		TransactionResult: &access.TransactionResult{
 			TransactionID: tx.ID(),
 			BlockID:       flow.ZeroID,
-			Status:        flow.TransactionStatusUnknown,
+			Status:        flow.TransactionStatusPending,
```

Comment:
this is true _if_ the caller of `SubscribeTransactionStatuses` also submitted a transaction. if we ever add endpoints that allow subscribing without sending a tx, this won't be the correct default.

since we know we'll be adding this for the websockets case, I think we should handle it now.